http.c

```diff
@@ -491,19 +490,7 @@ evhttp_maybe_add_date_header(struct evkeyvalq *headers)
 {
 	if (evhttp_find_header(headers, "Date") == NULL) {
 		char date[50];
```

Comment:
Note: the buffer length can be shortened to 30 bytes because the date string is always 29 bytes plus 1 byte null terminating string.